fdbserver/RESTSimKmsVault.actor.cpp

```diff
@@ -525,8 +547,9 @@ std::string getFakeBaseCipherIdsRequestContent(EncryptCipherDomainIdVec& domIds,
 	if (fault != FaultType::MISSING_VERSION) {
 		if (fault == FaultType::INVALID_VERSION) {
 			addVersionToDoc(doc, SERVER_KNOBS->REST_KMS_MAX_CIPHER_REQUEST_VERSION + 1);
+		} else {
+			addVersionToDoc(doc, SERVER_KNOBS->REST_KMS_MAX_CIPHER_REQUEST_VERSION);
 		}
-		addVersionToDoc(doc, SERVER_KNOBS->REST_KMS_MAX_CIPHER_REQUEST_VERSION);
```

Comment:
Why do we want to change the logic here? Is this a bug fix? Thanks

Reply:
Yes. We were adding version twice. Then the json was corrupt.